### ISSUE TITLE
Operator istio control plane tag not being updated

### ISSUE THREAD

**Issue by kramvan1** [istio/istio#18668]
**Bug description**

For `1.4.0-beta.2` the operators default control plane tag is still `1.4-dev`.
https://github.com/istio/operator/blob/1.4.0-beta.2/data/profiles/default.yaml#L5

**Expected behavior**

All profiles in https://github.com/istio/operator/tree/1.4.0-beta.2/data/profiles should be updated with the release tag.

Operator release would use the new Istio release image tag `1.4.0-beta.2` 

**Steps to reproduce the bug**

Install istio with the operator

**Version (include the output of `istioctl version --remote` and `kubectl version`)**

1.4.0 beta 2

**How was Istio installed?**

operator

**Environment where bug was observed (cloud vendor, OS, etc)**

**Comment by sdake** [comment]
@ostromart I am not really sure what to do about this bug.  I think it would make alot more sense to not hardcode this value, and instead retrieve it as part of the build and release process.  Any ideas?

**Comment by sdake** [comment]
@howardjohn this probably in some way involves the release builder code as well.  Any thoughts on how we can remove lines 4+5 from the profiles and have things just work would be valuable.

Cheers
-steve

**Comment by howardjohn** [comment]
not sure what happened to the other issue with the same info we just need to remove hub and tag from the profiles yaml so they use the defaults. Pure operator change

**Comment by howardjohn** [comment]
Nevermind, I'll fix it though

**Comment by kramvan1** [comment]
@howardjohn   This looks to still be broken, the default profile was missed with that PR above.
https://github.com/istio/operator/blob/release-1.4/data/profiles/default.yaml#L4-L6

We're still seeing `1.4-dev`  for the control plane images since we are using the default profile.

**Comment by howardjohn** [comment]
As mentioned, we do not support installing from github source

**Comment by howardjohn** [comment]
In the release we swap it out with the correct image

**Comment by kramvan1** [comment]
@howardjohn   Humm, we are not installing from the git hub source, we are using the operator image directly.     But since the default profile still has that tag in the operator image, we still are seeing the issue.

Or do I miss understand, there is no release operator image with this fix yet?    Using the `1.4-dev` won't work, right?

**Comment by sdake** [comment]
@howardjohn I had an older deploy of Istio even though I had thought I had deployed beta2.  I reset my K8s environment and had the following results:
```
sdake@beast-01:~$ curl -LO https://github.com/istio/istio/releases/download/1.4.0-beta.2/istio-1.4.0-beta.2-linux.tar.gz
...
sdake@beast-01:~$ tar -xzvf istio-1.4.0-beta.2-linux.tar.gz 
...
sdake@beast-01:~/istio-1.4.0-beta.2/install/kubernetes/operator$ kubectl apply -k deploy/
namespace/istio-operator created
customresourcedefinition.apiextensions.k8s.io/istiocontrolplanes.install.istio.io created
serviceaccount/istio-operator created
clusterrole.rbac.authorization.k8s.io/istio-operator created
clusterrolebinding.rbac.authorization.k8s.io/istio-operator created
service/istio-operator-metrics created
deployment.apps/istio-operator created
sdake@beast-01:~/istio-1.4.0-beta.2/install/kubernetes/operator$ kubectl describe po -n istio-operator istio-operator-899bffd88-669gq
...
  Normal  Pulling    50s   kubelet, beast-01  Pulling image "docker.io/istio/operator:1.4.0-beta.2"
  Normal  Pulled     44s   kubelet, beast-01  Successfully pulled image "docker.io/istio/operator:1.4.0-beta.2"
...
```

Looks like this is settled.

Thanks!
-steve
